src/models/employee.ts

```diff
@@ -0,0 +1,17 @@
+import { prop } from "@typegoose/typegoose";
+
+class Employee {
+  //? mongoDb has an intrinsic attribute _id do we also want employee Id?
+  //! Do we want to add password here?
```

Comment:
Let's not add passwords, we probably are going to be using their authentication and therefore won't need to store passwords